### ISSUE TITLE
Create a guide page for creating an event

### ISSUE THREAD

**Issue by arpansac** [commudle/commudle-ng#29]
We need a page to display how to use the event page, and other features, in the near future.

Once someone chooses the issue, I'll create a basic routed feature-module and you can start working on it.

**Comment by zackforbes** [comment]
I would like to go for this beginner issue .

**Comment by arpansac** [comment]
@zackforbes sure! I'll create a feature module for you and assign it to you by tomorrow. Till then please collect the steps for creating an event!

**Comment by zackforbes** [comment]
> @zackforbes sure! I'll create a feature module for you and assign it to you by tomorrow. Till then please collect the steps for creating an event!

steps should be for creating a new QSTP event ,  right ?

**Comment by arpansac** [comment]
Yup, could be for any community event.

I've added a new feature module 'help' at app/feature-modules/help. You can begin working on it.

The design can be inspired from: 
- https://developers.sendinblue.com/docs/getting-started
- https://help.videoask.com/hc/en-us

**Comment by zackforbes** [comment]
okay I'll start from today

**Comment by arpansac** [comment]
Here's another example: https://www.tiny.cloud/docs-4x/general-configuration-guide/upload-images/

**Comment by zackforbes** [comment]
> Here's another example: https://www.tiny.cloud/docs-4x/general-configuration-guide/upload-images/
> 
> Please keep communicating the progress here

okay Sir

**Comment by arpansac** [comment]
This issue is available to anyone who wants to do this.

**Comment by IshetaBansal** [comment]
Hey! I would love to work on this issue. Please assign!

**Comment by arpansac** [comment]
@IshetaBansal absolutely, have you set up the code on your system?

Also, we could take the way gitbook displays tutorials in reference for the design!

**Comment by IshetaBansal** [comment]
> @IshetaBansal absolutely, have you set up the code on your system?
> 
> Also, we could take the way gitbook displays tutorials in reference for the design!

Thanks a lot! 
Could you please guide me a little as this is my first issue?

**Comment by arpansac** [comment]
> > @IshetaBansal absolutely, have you set up the code on your system?
> > Also, we could take the way gitbook displays tutorials in reference for the design!
> 
> Thanks a lot!
> Could you please guide me a little as this is my first issue?

Yup, the current readme has the basic steps to get the app going on your system. Let me know if that doesn't work...!

**Comment by kanishkaa24** [comment]
Do we need to make a webpage as a guide ?

**Comment by arpansac** [comment]
We've implemented the help section within the application and would be revising the documentation soon again. Thanks for all the support!
